documenting/style-guide.rst

```diff
@@ -0,0 +1,223 @@
+.. _style-guide:
+
+=============
+ Style Guide
+=============
+
+.. highlight::  rest
+
+This document describes the style guide for our documentation.
+
+
+Use of whitespace
```

Comment:
```suggestion
Use of Whitespace
```

Reply:
The style guide currently uses a mix of "Sentence case" and "Title Case" for section headers.

Should we rather update the style guide to follow _itself_ and primarily use sentence case? :)

https://devguide.python.org/documenting/#capitalization